third_party/txt/tests/utils.cc

```diff
@@ -40,4 +46,74 @@ void SetCommandLine(fxl::CommandLine cmd) {
   gCommandLine = std::move(cmd);
 }
 
+void RegisterFontsFromPath(TypefaceFontAssetProvider& font_provider,
+                           std::string directory_path) {
+#if defined(_WIN32)
```

Comment:
I know you just moved these files, but we should replace the platform specific file handling routines with the ones in FML.

Reply:
Does FML offer a way to list files within a directory?

Reply:
If it doesn't, we should add it there (in a different patch is fine) so that all platforms can use it.